src/components/Drawer/mdc-temporary-drawer.vue

```diff
@@ -0,0 +1,114 @@
+<!-- thanks to petejohanson for this implementation -->
```

Comment:
sure I'm on board with avoiding capitals. Looking ahead, I think we'll need to have a README for each component/component group so folders will be helpful.